.travis.yml

```diff
@@ -12,6 +12,7 @@ install:
 jobs:
   include:
   - stage: Lint & Vendor Check
+    sudo: required
```

Comment:
Any implication of this like longer startup time? If it's like 10-20 seconds it does not matter anyway.